CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 
 ### 🛑 Breaking changes 🛑
 
+- `windowsperfcountersreceiver`: Added metrics configuration (#8376)
```

Comment:
Looks like I can't change this PR, so I can't fix this for you, but this now needs to be moved up to the "unreleased" section.

Reply:
Fixed